Documentation/Development.md

```diff
@@ -6,27 +6,232 @@
+:: terminate bazel daemon
+bazel --output_user_root %CD%/caches/bazel/tensorflow shutdown
+
+:: package
+set DESTDIR=%CD%\Library\tensorflow-windows-%VSCMD_ARG_TGT_ARCH%\tensorflow-2.4.0
+
+md %DESTDIR\usr\bin
+copy tensorflow\bazel-bin\tensorflow\tensorflow.dll %DESTDIR%\usr\bin\
+copy tensorflow\bazel-bin\tensorflow\compiler\tf2xla\xla_tensor\x10.dll %DESTDIR%\usr\bin\
+
+md %DESTDIR%\usr\lib
+copy tensorflow\bazel-out\%VSCMD_ARG_TGT_ARCH%_windows-opt\bin\tensorflow\tensorflow.lib %DESTDIR%\usr\lib\
+copy tensorflow\bazel-out\%VSCMD_ARG_TGT_ARCH%_windows-opt\bin\tensorflow\compiler\tf2xla\xla_tensor\x10.lib %DESTDIR%\usr\lib\
+
+md %DESTDIR%\usr\include\tensorflow\c
+copy tensorflow\tensorflow\c\c_api.h %DESTDIR%\usr\include\tensorflow\c\
+copy tensorflow\tensorflow\c\c_api_experimental.h %DESTDIR%\usr\include\tensorflow\c\
+copy tensorflow\tensorflow\c\tf_attrtype.h %DESTDIR%\usr\include\tensorflow\c\
+copy tensorflow\tensorflow\c\tf_datatype.h %DESTDIR%\usr\include\tensorflow\c\
+copy tensorflow\tensorflow\c\tf_file_statistics.h %DESTDIR%\usr\include\tensorflow\c\
+copy tensorflow\tensorflow\c\tf_status.h %DESTDIR%\usr\include\tensorflow\c\
+copy tensorflow\tensorflow\c\tf_tensor.h %DESTDIR%\usr\include\tensorflow\c\
+
+md %DESTDIR%\usr\include\tensorflow\c\eager
+cp tensorflow\tensorflow\c\eager\c_api.h %DESTDIR%\usr\include\tensorflow\c\eager\
+
+md %DESTDIR%\usr\include\x10
+copy swift-apis\Sources\x10\swift_bindings\device_wrapper.h %DESTDIR%\usr\include\x10\
+copy swift-apis\Sources\x10\swift_bindings\xla_tensor_tf_ops.h %DESTDIR%\usr\include\x10\
+copy swift-apis\Sources\x10\swift_bindings\xla_tensor_wrapper.h %DESTDIR%\usr\include\x10\
+
+md %DESTDIR%\usr\share
+copy tensorflow\bazel-out\%VSCMD_ARG_TGT_ARCH%_windows-opt\bin\tensorflow\tensorflow_filtered_def_file.def %DESTDIR%\usr\share
+```
+</details>
+
+<details>
+    <summary>macOS/Linux Build Script</summary>
+
+```bash
+# clone swift-apis
+git clone git://github.com/tensorflow/swift-apis
+# checkout tensorflow
+git clone --depth 1 --no-tags git://github.com/tensorflow/tensorflow
+git -C tensorflow checkout refs/tags/v2.4.0
+
+# copy X10
+cp -ar swift-apis/Sources/CX10 tensorflow/swift_bindings
+cp -ar swift-apis/Sources/x10/xla_client tensorflow/tensorflow/compiler/xla/xla_client
+cp -ar swift-apis/Sources/x10/xla_tensor tensorflow/tensorflow/compiler/tf2xla/xla_tensor
+# ensure that python dependencies are available
+python -m pip install --user numpy six
+# configure X10/TensorFlow
+export PYTHON_BIN_PATH=$(which python)
+export USE_DEFAULT_PYTHON_LIB_PATH=1
+export TF_NEED_OPENCL_SYCL=0
+export TF_DOWNLOAD_CLANG=0
+export TF_SET_ANDROID_WORKSPACE=0
+export TF_CONFIGURE_IOS=0 
+export TF_ENABLE_XLA=1
+export TF_NEED_ROCM=0
+export TF_NEED_CUDA=0
+export TF_CUDA_COMPUTE_CAPABILITIES=7.5
+export CC_OPT_FLAGS="-march=native"
+./tensorflow/configure.py
```

Comment:
Script error:
```
$ ./tensorflow/configure.py
./install-swift-apis: line 23: ./tensorflow/configure.py: Permission denied
```

Can this be replaced with running `./tensorFlow/configure`, which is a Bash script wrapper around `configure.py`?

Reply:
We could, but on the other hand, it is nice to keep as much symmetry between the different platforms.  Is there some particular reason you want to use the bash script wrapper?

Reply:
> Is there some particular reason you want to use the bash script wrapper?

Ah, I see. I just wanted to point out that running `./configure.py` directly doesn't work. Changing the line to `python3 ./tensorflow/configure.py` (so that it works) also sounds good!

Reply:
Ah, I missed that.  Thanks, I'll incorporate that.